lib/tool_shed/util/shed_util_common.py

```diff
@@ -512,6 +512,11 @@ def get_repository_for_dependency_relationship( app, tool_shed, name, owner, cha
                                                owner=owner,
                                                changeset_revision=changeset_revision )
     if not repository:
+        tool_shed_url = common_util.get_tool_shed_url_from_tool_shed_registry( app, tool_shed )
+        repository_clone_url = os.path.join( tool_shed_url, 'repos', owner, name )
+        repo_info_tuple = (None, repository_clone_url, changeset_revision, None, owner, None, None)
+        repository, pcr = repository_was_previously_installed( app, tool_shed_url, name, repo_info_tuple )
```

Comment:
@martenson Thanks!  I'll make sure to run those before changing any code and then submitting PRs that change Tool Shed code.  It will at least help to know that the same 12 break before changes and after.  ;)